packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -367,6 +420,18 @@ Resources:
               - ecr:GetAuthorizationToken
             Resource: "*"
             Effect: Allow
+          - Action:
```

Comment:
Does the image publishing role need key permissions? I can't find any documentation that indicates that it does.

Reply:
I also could not find anything and just now tried it locally and it seems to work without the permission. 👍🏿  thanks for the hint!